libs/sdk-ui-charts/styles/scss/tooltip.scss

```diff
@@ -1,28 +1,17 @@
 // (C) 2007-2020 GoodData Corporation
-@import "~@gooddata/goodstrap/lib/Bubble/styles/mixins";
-@import "~@gooddata/goodstrap/lib/core/styles/variables";
+@import "~@gooddata/sdk-ui-kit/styles/scss/variables";
+@import "~@gooddata/sdk-ui-kit/styles/scss/mixins";
 @import "variables";
 
 .gd-viz-tooltip {
     position: relative;
     border-radius: 3px;
     font-family: $font-stack-roman;
-    color: var(--gd-chart-backgroundColor-contrast, $gd-color-dark);
-    background-color: var(--gd-chart-backgroundColor-base-t05, transparentize($gd-color-white, 0.05));
-
-    .tt-name {
-        font-family: $font-stack-roman;
-        font-size: 12px;
-        word-wrap: break-word;
-        text-transform: uppercase;
-        color: #999;
-    }
-
-    .tt-tooltip {
-        margin-top: 2px;
-        margin-bottom: 14px;
-        font-size: 14px;
-    }
+    color: var(--gd-palette-complementary-6, $default-gd-color-dark);
+    background-color: var(
+        --gd-chart-tooltipBackgroundColor,
+        var(--gd-palette-complementary-0, transparentize($default-gd-color-white, 0.05))
+    );
```

Comment:
I'm missing previous variables as fallbacks here. Is it okay? Also do we need explicitly need `$default-gd-color-dark`?